packages/bigquery/src/index.js

```diff
@@ -175,6 +184,14 @@ BigQuery.prototype.dataset = function(id) {
  * }, callback);
  *
  * //-
+ * // If the callback is omitted, we'll return a Promise.
+ * //-
+ * bigquery.getDatasets()
```

Comment:
This is an example of how the docs will look when a single parameter is given back to the callback.